### PR TITLE
feat: upgrade Linux kernel to 5.7.14

### PR DESCRIPTION
**Pull request by smira**
This updates Linux kernel to the latest available version in 5.7.x
series.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: kernel/README.md [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
+make kernel-olddefconfig USERNAME=rsmitty PLATFORM=linux/arm64
 ```
 
 ## Customizing the kernel
 
 Run another target to get into `menuconfig`:
 
 ```sh
-make kernel-menuconfig
+make kernel-menuconfig USERNAME=rsmitty
```
Comment (andrewrynhard) [inline kernel/README.md:22]: .

Comment (smira) [reply kernel/README.md:22]: what does `.` mean? :)

Comment (andrewrynhard) [reply kernel/README.md:22]: Didn't know what else to say, but thought it would get your attention :smile:

Comment (andrewrynhard) [reply kernel/README.md:22]: I thought this was a typo.

Comment (smira) [reply kernel/README.md:22]: just docs, better to override

**Pull request merged**